packages/main/src/mainWindow.ts

```diff
@@ -31,6 +31,7 @@ async function createWindow() {
 
   const browserWindowConstructorOptions: BrowserWindowConstructorOptions = {
     show: false, // Use 'ready-to-show' event to show window
+    autoHideMenuBar: true, // This makes Podman Desktop look more like a native app
```

Comment:
I think that for this PR, we shouldn't address the menu bar item.
It's all about having a way to hide or exit on close

Reply:
Sure. I thought these we tightly coupled, so I figured we could address both in one PR, but that's fine. I'll remove this change.